src/app/docs/button/button-docs.component.html

```diff
@@ -161,28 +184,37 @@
       <sky-docs-design-guideline
       >
         <p>
-          For buttons with only one possible direct object such as <strong>Add</strong> buttons in record list toolbars, use the following format for button text:
+          For buttons with only one possible direct object such as <strong *skyThemeIf="'default'">Add</strong><strong *skyThemeIf="'modern'">New</strong> buttons in record list toolbars, use the following format for button text:
```

Comment:
I tried to serve locally so that I can see the updates while the PR checks aren't passing, and I get errors that say we can't bind `skyThemeIf` since isn't a known property of `strong`.